packages/table/src/types.tsx

```diff
@@ -0,0 +1,37 @@
+import { LidoComponentProps } from '@lidofinance/utils'
+export type { Theme } from '@lidofinance/theme'
+
+export enum LdTableAlign {
+  left,
+  center,
+  right,
+}
+export type LdTableAligns = keyof typeof LdTableAlign
+
+export enum LdTableTextColor {
+  primary,
+  secondary,
+  warning,
+  error,
+  success,
+}
+export type LdTableTextColors = keyof typeof LdTableTextColor
+
+export type LdTableProps = LidoComponentProps<'div'>
+export type LdTheadProps = LidoComponentProps<'div'>
+export type LdTbodyProps = LidoComponentProps<'div'>
+export type LdTfootProps = LidoComponentProps<'div'>
+export type LdTrProps = LidoComponentProps<'div'>
+export type LdThProps = LidoComponentProps<
+  'div',
+  {
+    align?: LdTableAligns
+  }
+>
+export type LdTdProps = LidoComponentProps<
+  'div',
```

Comment:
Maybe we should use standard table elements (table, thead, tbody, tr, td) instead of divs? Table elements are semantically correct and they guarantee column alignment, when the cell content overflows:

![image](https://user-images.githubusercontent.com/1422379/130361405-d5298197-75e6-49b5-97d9-f10d8f3ec801.png)

Also, standard elements have functionality that is not so easy to repeat on flexboxes: `rowspan` and `colspan`, for example. An HTML table can be copied and pasted into a google docs or excel. And table has great accessibility

Reply:
need to discuss.